bay-services/gremlin.yaml

```diff
@@ -15,3 +16,21 @@ services:
       REPLICAS: 1
   path: /openshift/template.yaml
   url: https://github.com/fabric8-analytics/gremlin-docker/
+
+- <<: gremlin_def:
```

Comment:
I think the colon at the end causes the parsing error.

Reply:
Fixed.